File: 7KYU/Sum of integers in string/solution.cs
using System;
using System.Linq;
using System.Text.RegularExpressions;
public class Kata
{
  public static int SumOfIntegersInString(string s)
  {
      var matches = Regex.Matches(s, @"\d+");
      var total = 0;
      foreach(Match match in matches)
      {
          GroupCollection groups = match.Groups;
          total += int.Parse(match.Value.ToString());
      }
      return total;  
  }
}

File: 7KYU/Sum of integers in string/solution.go
package kata

import (
	"regexp"
	"strconv"
)

func SumOfIntegersInString(strng string) int {
	re := regexp.MustCompile(`\d+`)

	matches := re.FindAllString(strng, -1)
	res := 0
	for _, match := range matches {
		n, _ := strconv.Atoi(match)
		res += n
	}
	return res
}
